src/renderer/common/components/dialog/publicationInfos/PublicationInfoA11y2.tsx

```diff
@@ -161,8 +161,9 @@ export const PublicationInfoA11y2: React.FC<IProps> = ({publicationViewMaybeOpds
     const tactile_graphic = findStrInArray(a11y_accessibilityFeature, "tactileGraphic");
     const tactile_object = findStrInArray(a11y_accessibilityFeature, "tactileObject");
     const text_to_speech_hinting = findStrInArray(a11y_accessibilityFeature, "ttsMarkup");
-    const print_page_numbers = findStrInArray(a11y_accessibilityFeature, "printPageNumbers"); // TODO: already declared in previous a11y implementation but not in https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html // declared in readium webpub manifest https://github.com/readium/webpub-manifest/blob/4c73f7323f9241e61bb919ecae2656a491ba15f6/schema/a11y.schema.json#L84
-
+    // const print_page_numbers = findStrInArray(a11y_accessibilityFeature, "printPageNumbers"); 
+    // Replaced by pageBreakMarkers
```

Comment:
you have to remove it in a11y.schema.json in this case. 

https://github.com/readium/webpub-manifest/blob/master/schema/a11y.schema.json#L84

Reply:
Deprecation doesn't necessarily mean removal though ... in fact reading systems / processing agents still have to interpret the old metadata name present in legacy publications...

Reply:
Many Thanks @gautierchomel

Reply:
@danielweck https://github.com/edrlab/thorium-reader/pull/2937/commits/d6dd1bd0fcd2445e1f4101a5e48f07684bcb70cd